docs/LOGS.md

```diff
@@ -0,0 +1,130 @@
+| `"request": {`            | field regarding request information                                                                                          |
+| &nbsp;&nbsp;`"bytes"`     | body size of request                                                                                                         |
+| &nbsp;&nbsp;`"headers"`   | field regarding keys and values originating from configured keys/header names                                                |
+| &nbsp;&nbsp;`"host"`      | host of request                                                                                                              |
+| &nbsp;&nbsp;`"method"`    | http request method, see [Mozilla Reference](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods) for more information |
+| &nbsp;&nbsp;`"origin"`    | origin of request                                                                                                            | 
+| &nbsp;&nbsp;`"path"`      | path of request                                                                                                              |
+| &nbsp;&nbsp;`"proto"`     | protocol of request                                                                                                          |
+| &nbsp;&nbsp;`"status"`    | status of request                                                                                                            | 
+| &nbsp;&nbsp;`"tls" }`     | TLS used `true` or `false`                                                                                                   |
+| `"response": {`           | field regarding response information                                                                                         |
+| &nbsp;&nbsp;`"bytes"`     | body size of response                                                                                                        |
+| &nbsp;&nbsp;`"headers" }` | field regarding keys and values originating from configured keys/header names                                                |
+| `"server"`                | server name (defined in couper file)                                                                                         |
+| `"status"`                | response status code                                                                                                         |
+| `"timings": {`            | field regarding timing                                                                                                       |
+| &nbsp;&nbsp;`"total" }`   | total time taken                                                                                                             |
+| `“uid"`                   | unique request id configurable in [Settings](../docs/REFERENCE.md#settings-block)                                            |
```

Comment:
Applies to all `../docs/*` appearances: I think `./*` would be enough since this document lies next to the referenced ones.